src/wp-includes/fonts.php

```diff
@@ -188,3 +188,13 @@ function _wp_before_delete_font_face( $post_id, $post ) {
 		wp_delete_file( $font_dir . '/' . $font_file );
 	}
 }
+
+/**
+ * Register the default font collections.
+ *
+ * @access private
+ * @since 6.5.0
+ */
+function _register_font_collections() {
+	wp_register_font_collection( 'google-fonts', 'https://s.w.org/images/fonts/17.7/collections/google-fonts-with-preview.json' );
+}
```

Comment:
From an extenders perspective do we anticipate that in the future there might be more default collections in additional to Google Fonts?

If so then should we consider a way to allow extenders to filter which default collections are/aren't registered.

Say for example I don't want Google fonts on my site by default, how can I unhook this collection without also unhooking any other collections that might be registered here?

Reply:
There's an "unregister_font_collection" function for this use case.

Reply:
Ok it seems like we have sufficient opt routes in place 👍

Reply:
Also, don't forget that this is all hooked in `default-filters.php`, which is conventional and gives extenders the last-resort option to remove that action.